app/src/main/java/com/elavon/converge/model/mapper/ConvergeMapper.java

```diff
@@ -550,24 +550,25 @@ public void mapTransactionResponse(final ElavonTransactionResponse etResponse, f
             processorResponse.setApprovalCode(etResponse.getApprovalCode());
         }
 
+        if(processorResponse != null && !StringUtil.isEmpty(etResponse.getAmount())) {
```

Comment:
processor response should never be null here right?  it's instantiated at the beginning of this method

Reply:
yes correct, no need of null check for processorResponse object.